### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by kyleam**
Most of these are minor, but there may be objections to the changes in the last two commits.

### CODE REVIEW

**Comment by adswa** [comment]
I like all of the changes :+1:

File: paper.md [hunk 1]
```diff
@@ -227,13 +228,13 @@ Such annotation is not sufficient to introduce changes by following the descript
...
 Git-annex readily provides access to a wide range of external data storage resources via a large set of protocols.
 DataLad builds on this support and adds, for example, more fine-grained access (e.g.
-direct access to individual components contained in an archive hosted on cloud storage), or specialized services, such as XNAT ([www.xnat.org](http://www.xnat.org)).
+direct access to individual components contained in an archive hosted on cloud storage) or specialized services, such as XNAT ([www.xnat.org](http://www.xnat.org)).
 Efficient and seamless access to scientific data is implemented using
-the *special remote* protocol provided by [@git-annex:special_remotes_protocol], through which external tools, like DataLad, can provide custom transport functionality transparently to a git-annex user.
+the *special remote* protocol provided by [@git-annex:special_remotes_protocol], through which external tools, like DataLad, can provide custom transport functionality transparently to a git-annex user.<!-- km: The citation in this sentence appears in parentheses: "by (Hess 2013)," -->
```
Comment (mih) [inline paper.md]:
```suggestion
the *special remote* protocol provided by git-annex [@git-annex:special_remotes_protocol], through which external tools, like DataLad, can provide custom transport functionality transparently to a user.
```

**Review by mih — APPROVED** [review]
Thx much! I support this motion in full -- I left one suggestion to resolve the `()` issue

File: paper.md [hunk 2]
```diff
@@ -168,27 +168,26 @@ In order to maximize its utility and target audience, DataLad is available for a
...
 This makes it hard or impossible to use Git directly for distributed data storage with tailored access to individual files.
-Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.
+Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.<!-- km: I think this sentence could be dropped. -->
 File content handled by git-annex is placed into a managed repository annex, instead of being committing to Git directly.
-Rather than the actual file content, git-annex only commits a compact reference that enables identification and association of a file name with the content.
-Those file content references are typically based on a checksum of the content.
-Using these identifiers, git-annex tracks content availability across all repository clones (local or remote), or external resources such as URLs pointing to individual files on the web.
+Instead git-annex commits a compact reference, typically derived from the checksum of a file's content, that enables identification and association of a file name with the content.
```
Comment (mikapfl) [inline paper.md]:
```suggestion
Instead, git-annex commits a compact reference, typically derived from the checksum of a file's content, that enables identification and association of a file name with the content.
```

Comment (mikapfl) [reply paper.md]: Apart from the missing comma, there is a doubling of "instead" with the previous sentence, maybe the previous sentence should drop the "instead" and use something like "…, which avoids committing the file content directly to Git."

**Comment by yarikoptic** [comment]
Looks good to me too @kyleam , thank you . The double instead @mikapfl mentioned might have been intentional, but it does throw me off a bit too.

File: paper.md (hunk 2 above)
Comment (kyleam) [reply paper.md]: > here is a doubling of "instead" with the previous sentence, maybe the previous sentence should drop the "instead" and use something like "…, which avoids committing the file content directly to Git."

True, I didn't notice that.  I'll go with your suggestion.  Thanks.

**Pull request merged**